source/shared/core_sqlsrv.h

```diff
@@ -1889,14 +1889,20 @@ namespace core {
         // and return a more helpful message prepended to the ODBC errors if that error occurs
         if( !SQL_SUCCEEDED( r )) {
 
-            SQLCHAR err_msg[SQL_MAX_ERROR_MESSAGE_LENGTH + 1] = {'\0'};
+            SQLCHAR err_msg[SQL_MAX_MESSAGE_LENGTH + 1] = {'\0'};
             SQLSMALLINT len = 0;
             
             SQLRETURN rtemp = ::SQLGetDiagField( stmt->handle_type(), stmt->handle(), 1, SQL_DIAG_MESSAGE_TEXT, 
-                                             err_msg, SQL_MAX_ERROR_MESSAGE_LENGTH, &len );
+                                             err_msg, SQL_MAX_MESSAGE_LENGTH, &len );
```

Comment:
So it's not necessary to use SQL_MAX_ERROR_MESSAGE_LENGTH here?

Reply:
Yes, because the sole purpose of this check is to compare the error message to ODBC_CONNECTION_BUSY_ERROR, which is very short